api_tester/lib/api_tests/purchases_flutter_api_test.dart

```diff
@@ -72,42 +72,49 @@ class _PurchasesFlutterApiTest {
   void _checkPurchaseProduct() async {
     String productIdentifier = "fakeProductId";
     UpgradeInfo? upgradeInfo;
+    GoogleProductChangeInfo? googleProductChangeInfo;
     PurchaseType purchaseType = PurchaseType.subs;
     CustomerInfo customerInfo = await Purchases.purchaseProduct(
         productIdentifier,
         upgradeInfo: upgradeInfo,
-        isPersonalizedPrice: true);
+        googleIsPersonalizedPrice: true);
```

Comment:
hmm i wonder if we can remove the google prefix now that these are within the GoogleProductChangeInfo class? to make it a little less wordy

Reply:
@beylmk `googleIsPersonalizedPrice` isn't in the `GoogleProductChangeInfo` but... should it be? 🤔 It can be used without product change though, right?